From ab031c2b1f8c03e23a8dc8a95c9c9e9b8ce397b2 Mon Sep 17 00:00:00 2001
From: Mingli Yu <mingli.yu@windriver.com>
Date: Fri, 15 Sep 2023 11:32:11 +0800
Subject: [PATCH] BundledOSSPUUID.cmake: Pass CFLAGS to compiler

Make sure -fdebug-prefix-map options are passed to compiler to fix the
below warning:
  WARNING: minifi-cpp-0.7.0-r0 do_package_qa: QA Issue: File /usr/bin/.debug/minificontroller in package minifi-cpp-dbg contains reference to TMPDIR
File /usr/bin/.debug/minifi in package minifi-cpp-dbg contains reference to TMPDIR [buildpaths]

Upsteam-Status: Pending

Signed-off-by: Mingli Yu <mingli.yu@windriver.com>
---
Upstream-Status: Pending

 cmake/BundledOSSPUUID.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/BundledOSSPUUID.cmake b/cmake/BundledOSSPUUID.cmake
index 0cf4bac0..ec57ef51 100644
--- a/cmake/BundledOSSPUUID.cmake
+++ b/cmake/BundledOSSPUUID.cmake
@@ -37,7 +37,7 @@ function(use_bundled_osspuuid SOURCE_DIR BINARY_DIR)
     ENDFOREACH(BYPRODUCT)
 
     # Build project
-    set(CONFIGURE_COMMAND ac_cv_va_copy=C99 ./configure CFLAGS=-fPIC CXXFLAGS=-fPIC --host=${HOST_SYS}
+    set(CONFIGURE_COMMAND ac_cv_va_copy=C99 ./configure CXXFLAGS=-fPIC --host=${HOST_SYS}
         --with-cxx --without-perl --without-php --without-pgsql
         --prefix=${BINARY_DIR}/thirdparty/ossp-uuid-install)
     string(TOLOWER "${CMAKE_BUILD_TYPE}" build_type)
-- 
2.25.1

